tests/Functional/Controller/Admin/LocationControllerTest.php

```diff
@@ -70,20 +62,36 @@ public function testIndexActionNotAdmin(): void
 
     public function testNewAction(): void
     {
+        $name = 'here';
         $address = 'testname';
 
         // Act
         $crawler = $this->client->request('GET', '/admin/location/new');
 
         // Act
         $form = $crawler->selectButton('Toevoegen')->form();
+        $form['location[name]'] = $name;
         $form['location[address]'] = $address;
 
         $crawler = $this->client->submit($form);
         self::assertEquals(200, $this->client->getResponse()->getStatusCode());
         self::assertSelectorTextContains('.container', 'Locatie '.$address);
     }
 
+    public function testMergeAction(): void
```

Comment:
I think this test would be improved if it would also verify that activity references were updated as well.